src/shared/components/oncoprint/TooltipUtils.ts

```diff
@@ -97,18 +98,21 @@ export function makeHeatmapTrackTooltip(genetic_alteration_type:MolecularProfile
         let data_header = '';
         let profile_data = 'N/A';
         switch(genetic_alteration_type) {
-            case "MRNA_EXPRESSION":
+            case AlterationTypeConstants.MRNA_EXPRESSION:
                 data_header = 'MRNA: ';
                 break;
-            case "PROTEIN_LEVEL":
+            case AlterationTypeConstants.PROTEIN_LEVEL:
                 data_header = 'PROT: ';
                 break;
-            case "METHYLATION":
+            case AlterationTypeConstants.METHYLATION:
                 data_header = 'METHYLATION: ';
                 break;
+            case AlterationTypeConstants.TREATMENT_RESPONSE:
+                data_header = 'TREATMENT: ';
+                break;
         }
         if ((d.profile_data !== null) && (typeof d.profile_data !== "undefined")) {
-            profile_data = d.profile_data.toFixed(2);
+            profile_data = d.category || d.profile_data.toFixed(2);
```

Comment:
this is really specific to TREATMENT_RESPONSE - you should do a check `if(TREATMENT_RESPONSE) { // what you have (line 115) } else {  // what it was before (line 111) }`

Reply:
Ok, added a condition for treatment data.